### PR TITLE
Bug 1479700 - IntegrationTests Update dependencies

### PR DESCRIPTION
**Pull request by isabelrios**
This PR is to fix the failing jobs when running Integration Tests due to dependencies issues

### CODE REVIEW

File: SyncIntegrationTests/Pipfile.lock [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+            "os_name": "posix",
+            "platform_machine": "x86_64",
+            "platform_python_implementation": "CPython",
+            "platform_release": "17.7.0",
+            "platform_system": "Darwin",
+            "platform_version": "Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64",
+            "python_full_version": "2.7.14",
+            "python_version": "2.7",
+            "sys_platform": "darwin"
+        },
```
Comment (davehunt) [inline SyncIntegrationTests/Pipfile.lock]: This suggests you're using an old version of pipenv. Could you update and run `pipenv update` again? If you installed using Homebrew you can run `brew update && brew upgrade`.

File: Jenkinsfile [hunk 2]
```diff
@@ -22,7 +22,7 @@ pipeline {
             steps {
                 dir('SyncIntegrationTests') {
                     sh 'pipenv install'
-                    sh 'pipenv check'
+                    sh 'pipenv check -i 36351'
```
Comment (davehunt) [inline Jenkinsfile]: Could you add a comment mentioning https://github.com/pyupio/safety-db/issues/2272

File: SyncIntegrationTests/Pipfile.lock (hunk 1 above)
Comment (isabelrios) [reply SyncIntegrationTests/Pipfile.lock]: @davehunt I run those commands but do not see any difference on Pipfile.lock
I have pipenv 2018.7.1 which I think is the latest

File: Jenkinsfile (hunk 2 above)
Comment (isabelrios) [reply Jenkinsfile]: Ah sorry forgot you mentioned it, just added it

File: SyncIntegrationTests/Pipfile.lock (hunk 1 above)
Comment (davehunt) [reply SyncIntegrationTests/Pipfile.lock]: hmm.. I wonder why it's not clearing this section. could you try removing this file and recreating it with `pipenv lock`?

Comment (isabelrios) [reply SyncIntegrationTests/Pipfile.lock]: 👍

Comment (isabelrios) [reply SyncIntegrationTests/Pipfile.lock]: I can still see that line after re-creating the Pipfile.lock but some hashes have changed, @davehunt is it still wrong? 
Thanks!

Comment (isabelrios) [reply SyncIntegrationTests/Pipfile.lock]: No idea why it is still shown :( but agree, does not seem to affect the tests, they are green in this PR. Thanks! enjoy your PTO!

**Pull request merged**